vim-canvas-demo-app-react/src/components/patient-enhancements/PatientEnhancements.tsx

```diff
@@ -80,6 +88,17 @@ export const PatientEnhancements = () => {
             `Unknown enhancement function: ${enhancement.function}`
           );
         }
+      } catch (error) {
+        const errorMessage =
+          (error as { code?: number }).code === 422 //not supported
```

Comment:
I think a 422 error is too general. We need some protocol with the client about specific error messages, something like throw new Error('ACTION_NOT_SUPPORTED_BY_EHR')